src/components/utils/test/thread_test.cc

```diff
@@ -175,6 +247,175 @@ TEST_F(ThreadTest, StartOneThreadTwice_ExpectTheSameThreadStartedTwice) {
   EXPECT_TRUE(delegate_->check_value());
 }
 
+TEST(ThreadTest2, StartAndJoinForce_Success) {
```

Comment:
@AGaliuzov @AGritsevich It states that it's the second iteration of tests for the threads. Also this is not the name of the test but the name of the object that is being tested. I can't use `ThreadTest`. I don't think these suggestion reflect the nature of the tests.